### DISCUSSION TITLE
TypeError: require.resolveWeak is not a function (jest test)

### DISCUSSION THREAD

**Discussion by Emiliano-Bucci** [vercel/next.js#19629] [Help]
It seems that there's a bug when trying to test a component which contains a dynamic imported component. The error i get is `TypeError: require.resolveWeak is not a function`.

I prepare a repo example -> https://github.com/Emiliano-Bucci/jest-with-dynamic-import which is basically the `with-jest` example to which i add the dynamic component to test and reproduce the error. 

**Original discussion**: https://github.com/vercel/next.js/discussions/18855#discussioncomment-131093

@lfades Hi! This is the third issue i open about this, and every time is being moved here to discussions, while this is a bug :/ Why is this happening? :)

**Comment by lfades** [comment]
@Emiliano-Bucci It was probably moved because you need to follow the issue template to create issues, it's required to follow the issue template 🙏
